pages/image/image.css

```diff
@@ -0,0 +1,17 @@
+.square-img {
+  width: 150px;
+  height: 150px;
+  border-radius: 5px;
```

Comment:
Hey Kevin, just a suggestion, the image looks kind of distorted here. You can use `object-fit: cover;`  so the image will fill the height and width of its box.

Reply:
Done !!